src/glb-healthcheck/main.go

```diff
@@ -48,6 +48,11 @@ var (
 	HealthCheckInterval = 2 * time.Second
 )
 
+// Default values for the marking-thresholds for marking backends (un)healthy
+var (
+	SuccessesBeforeMarkedHealthy = 3
+	FailuresBeforeMarkedFailed   = 3
```

Comment:
It initially confused me that we were writing to these global defaults (and I should have made them `const` originally too!). Perhaps it could be clearer to make these `const` instead of `var` then name these e.g. `DefaultSuccessesBeforeMarkedHealthy` then set locals below as `successesBeforeMarkedHealthy := DefaultSuccessesBeforeMarkedHealthy` then override if set.

Reply:
Sure.